crates/holochain_types/src/app.rs

```diff
@@ -975,9 +978,7 @@ impl StoppedAppReason {
         match status {
             AppStatus::Paused(reason) => Some(Self::Paused(reason.clone())),
             AppStatus::Disabled(reason) => Some(Self::Disabled(reason.clone())),
-            AppStatus::AwaitingMemproofs => {
-                Some(Self::Disabled(DisabledAppReason::AwaitingMemproofs))
-            }
+            AppStatus::AwaitingMemproofs => Some(Self::AwaitingMemproofs),
```

Comment:
There's no way back into this state right? So one we've gone through this flow of providing mem-proofs and enabling the app on the app interface. If the app gets enabled or the conductor restarts - there is then no way to back to a state where the app interface can use this route to enable the app again?

Reply:
Yes, the state transition function has no transitions to this state (except from itself) https://github.com/holochain/holochain/blob/b3c7f90435c70922979f75d76bb8761ef7a0957f/crates/holochain_types/src/app.rs#L875-L903, and the only way to put an app in this state is to create it that way from the beginning.